LICENSE.md

```diff
@@ -0,0 +1,660 @@
+### GNU AFFERO GENERAL PUBLIC LICENSE
+
+Version 3, 19 November 2007
+
+Copyright (C) 2007 Free Software Foundation, Inc.
+<https://fsf.org/>
```

Comment:
Is this the license of the package itself or the app? If it's of the package, it's okay to keep it otherwise I don't think it's needed here.

Reply:
Both Repomaker and its flatpak package are licensed below AGPL-3.0, so I think it's OK to keep it here.